urbs.py

```diff
@@ -1894,18 +1895,27 @@ def report(instance, filename, commodities=None, sites=None):
             for sit in sites:
                 created, consumed, stored, imported, exported, derivative = get_timeseries(
                     instance, co, sit)
+                
+                # temporarily drop out the timeseries demand and delta, for a correct overproduction calculation
```

Comment:
I'm a bit unsure whether this approach "scales" well. In my books, the expression that calculates `overprod` should always hold, not only after removing two specially named columns. In other words: the uninitiated observer looking at the resulting report spreadsheet has no way of inferring - from the visible structure alone - how the column overproduction was calculated. He/she must instead analyse the meanings of all columns and derive that two (demand, demand-delta) columns are special and not part of the balance.